IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormInitTests.cs

```diff
@@ -65,7 +65,6 @@ public void Should_show_current_GitModuleWorkingDir_if_supplied_path_null()
         [TestCase("")]
         [TestCase(null)]
         [TestCase("    ")]
-        [TestCase(@"c:\foo*\bar?\")]
```

Comment:
💡 If you switch to using `GetInvalidPathChars` (per a previous comment), it may make sense to restore several of the test inputs that are removed.

Reply:
I don't want to tinker with this in the context of the PR. I'll raise a new issue once this is merged.

Reply:
Does `TestCase` have a `Skip` property (like `Fact` has in XUnit)?

Reply:
See above. These are Windows/NTFS-ish specific use cases.